packages/core/src/telemetry/service.ts

```diff
@@ -15,7 +15,7 @@ import { postEvent } from "@/telemetry/post-event";
 import { getGitRemoteUrl } from "@/telemetry/remote";
 
 type TelemetryEvent = {
-  eventName: string;
+  event: string;
```

Comment:
right now this doesn't make any sense but Segment requires an `event` field for their track events. This doesn't mean we're tied to Segment but doesn't hurt to set it